src/CMakeLists.txt

```diff
@@ -59,7 +59,6 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmark_version.h.in
 include (GenerateExportHeader)
 
 add_executable(${PROGRAM} ${PROGRAM_SOURCES})
-add_compiler_export_flags()
```

Comment:
Can you explain why this line is removed?

Reply:
The function is deprecated see commit message.